Add "How To Get Your Pull Request Reviewed?" to developer handbook (#40589)
WordPress/gutenberg: docs/contributors/how-to-get-your-pull-request-reviewed.md

# How To Get Your Pull Request Reviewed?

> Sometimes we publish a Pull Request and noone reviews our work. What to do?

Reviewer: ```suggestion
Sometimes we publish a Pull Request and no one reviews our work. What to do?
```

Attracting a review largely isn't about the code – it is about making the reviewing easy.

> If you published a Pull Request that isn't getting any comments or revies, try one of the strategies used by core contributors:

Reviewer: ```suggestion
If you published a Pull Request that isn't getting any comments or reviews, try one of the strategies used by core contributors:
```

## Create the smallest reasonable PRs

Approving a 2000-line-long PR takes months and feels overwhelming.

Approving a 50-line long PR takes days or hours and feels easy.

Large batches slow you down. Ship your work in small chunks to merge more and learn faster.

> ## Share ALL the context:

Reviewer 2: As mentioned in this other PR, the word `all` might be too much. I'd clarify with something along the lines of "all the context needed".

Reviewer: ```suggestion
## Share relevant context:
```

How about this?

> Say:

Reviewer: ```suggestion
Clarify:
```

* What problem are you solving?
* How does your PR solve it?
* What feedback do you need?
* What’s out of scope?
* What’s unintuitive?
* How to test?

Summarize any related issues and PRs.

It's easier than asking others to go and figure it out.

## Make your PR exciting

All contributions are competing for attention. Make your stand out.

The easiest way? Say why it matters:

❌ A new react hook to get data

> ✅ useEntityRecord: get data with 10x less boilerplate

Reviewer 2: ```suggestion
✅ `useEntityRecord`: get data with 10x less boilerplate
```

Then prove it with code examples, visuals, and screencasts.

## Show your work

Post a link to your PR in related issues & PRs.

Ping commenters of related issues, previous committers, and tech leads.

> Bring it up on WP.org slack.

Reviewer: ```suggestion
Bring it up on [WP.org slack](https://make.wordpress.org/core/tag/core-editor-agenda/).
```

Assign relevant labels, milestones, and projects (or ask someone).

## Review the work of others

It’s the easiest way to get on others’ radar.

> Look up the PRs of folks listed in the previous tweet and review them.

Reviewer 2: The tweet reference here is out of context.

Is their work unfamiliar? Do:

* Take some time to understand it
* Propose a pair programming session
* Skip, go for the next PR

> ## Grease risk with clarity

Reviewer: ```suggestion
## Reduce risk with clarity
```

"Grease" wasn't immediately clear to me. We can probably simplify it a bit.

Risk adds friction – an approval can backfire later.

Clarity is like grease. Clearly document:

* What risks are involved? Why take them?
* Why is this PR the best solution?
* How can the risk be minimized?
* What else has been tried?

## Follow the attention

Some PRs naturally get more traction than others.

Double down on these.

> It’s like finding a product-market fit as a startup; only you’re after the idea–reviewers fit.

Reviewer 2: This is hard for me to understand in the context of the handbook. Is there a different analogy? Could you elaborate on this?

Reviewer: I think you're looking to say ...

```suggestion
Some Issues are more topical than others (e.g. those listed in the goals for an upcoming release) and thus will garner more attention. By focusing on these it will be easier to attract reviewers.
```
